gradle example fails to build

I cloned and built wildfly-swarm locally (which now produces version 1.0.0.Alpha5-SNAPSHOT) and then update the build.gradle in wildfly-swarm-example/gradle to point to the same version.

I run the following (on my CentOS 6.6 development environment):
```
JAVA_HOME=/opt/devtools/jdk1.8.0_60 ./gradlew wildfly-swarm-package
```
which, after resolving any dependencies, results in these compile errors:
```
:compileJava
/tmp/wildfly-swarm-examples/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java:4: error: cannot find symbol
import org.wildfly.swarm.jaxrs.JAXRSDeployment;
                              ^
  symbol:   class JAXRSDeployment
  location: package org.wildfly.swarm.jaxrs
/tmp/wildfly-swarm-examples/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java:5: error: cannot find symbol
import org.wildfly.swarm.undertow.WarDeployment;
                                 ^
  symbol:   class WarDeployment
  location: package org.wildfly.swarm.undertow
/tmp/wildfly-swarm-examples/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java:17: error: cannot find symbol
        WarDeployment war = new JAXRSDeployment(container);
        ^
  symbol:   class WarDeployment
  location: class Main
/tmp/wildfly-swarm-examples/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java:17: error: cannot find symbol
        WarDeployment war = new JAXRSDeployment(container);
                                ^
  symbol:   class JAXRSDeployment
  location: class Main
4 errors
:compileJava FAILED
```

I'm not really sure if these classes have been replaced or something as I can't seem to find them by google search or by looking in the jars in my local .m2 repos.  Any chance this example is just out-of-date and needs to be updated?

--Dann

--- a/gradle-mail/build.gradle
+++ b/gradle-mail/build.gradle
@@ -1,5 +1,5 @@
 buildscript {
-  version = '1.0.0.Alpha4-SNAPSHOT'
+  version = '1.0.0.Alpha5-SNAPSHOT'
   repositories {
     mavenLocal()
   }

--- a/gradle-mail/src/main/java/org/wildfly/swarm/examples/mail/Main.java
+++ b/gradle-mail/src/main/java/org/wildfly/swarm/examples/mail/Main.java
@@ -1,10 +1,10 @@
 package org.wildfly.swarm.examples.mail;
 
+import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.wildfly.swarm.container.Container;
-import org.wildfly.swarm.jaxrs.JAXRSDeployment;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
 import org.wildfly.swarm.mail.MailFraction;
 import org.wildfly.swarm.mail.SmtpServer;
-import org.wildfly.swarm.undertow.WarDeployment;
 
 /**
  * @author Helio Frota
@@ -22,10 +22,11 @@ public class Main {
                 .host("add_your_smtp_here")
                 .port("25")));
         
-        WarDeployment deployment = new JAXRSDeployment(container);
-        deployment.getArchive().addClass(Mail.class);
+        JAXRSArchive deployment = ShrinkWrap.create(JAXRSArchive.class);
+        deployment.addClass(Mail.class);
+        deployment.addAllDependencies();
         container.start().deploy(deployment);
-        
+
         System.out.println(smtp.jndiName());
 
     }

--- a/gradle/build.gradle
+++ b/gradle/build.gradle
@@ -1,5 +1,5 @@
 buildscript {
-  version = '1.0.0.Alpha4-SNAPSHOT'
+  version = '1.0.0.Alpha5-SNAPSHOT'
   repositories {
     mavenLocal()
   }

--- a/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java
+++ b/gradle/src/main/java/org/wildfly/swarm/examples/gradle/Main.java
@@ -1,8 +1,8 @@
 package org.wildfly.swarm.examples.gradle;
 
+import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.wildfly.swarm.container.Container;
-import org.wildfly.swarm.jaxrs.JAXRSDeployment;
-import org.wildfly.swarm.undertow.WarDeployment;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
 
 /**
  * @author helio frota
@@ -14,10 +14,10 @@ public class Main {
 
         Container container = new Container();
 
-        WarDeployment war = new JAXRSDeployment(container);
-        war.staticContent("/");
-        war.getArchive().addClass(HelloRest.class);
-        container.start().deploy(war);
+        JAXRSArchive deployment = ShrinkWrap.create(JAXRSArchive.class);
+        deployment.addClass(HelloRest.class);
+        deployment.addAllDependencies();
+        container.start().deploy(deployment);
 
     }
 }

